content/en/blog/grpc-csharp-future.md

```diff
@@ -204,27 +93,28 @@ on how to migrate from Grpc.Core to grpc-dotnet.
 We plan to publish a more detailed migration guide in the future to facilitate
 the migration from Grpc.Core to grpc-dotnet.
 
-**I'm planning to use gRPC in C# in my future project. Which implementation should I choose?**
+### I'd like to use gRPC in C# for a new project. Which implementation should I choose?
 
 We strongly recommend only using grpc-dotnet for new projects. We are going to
 stop supporting Grpc.Core in the future.
 
-**Does this mean I need to stop using Grpc.Core right now?**
+### Does this mean I need to stop using Grpc.Core right now?
 
 No, Grpc.Core will continue to be supported for a while (see the deprecation
 schedule). There should be enough time for you to assess the situation and plan
 your migration.
 
-**I'm not using gRPC directly in my code, but I'm using the Google Cloud Client Libraries (which do use Grpc.Core under the hood). How does this impact me?**
+### I'm using Google Cloud Client Libraries, does this impact me?
 
-This deprecation does not currently impact existing users of Google Cloud Client
-Libraries.
+Google Cloud Client Libraries use Grpc.Core under the hood. The Grpc.Core
```

Comment:
as mentioned offline, we need to be careful with rewordings in this particular question. I'd rather leave this question as is (or maybe made updates in a separate PR).